examples/metal-nimf-2-oracle-connection/main.tf

```diff
@@ -78,3 +78,12 @@ module "metal_2_oracle_connection" {
   zside_seller_region         = var.oracle_region
   zside_fabric_sp_name        = var.zside_fabric_sp_name
 }
+
+resource "time_sleep" "wait_dl_connection" {
+  depends_on = [module.metal_2_oracle_connection]
+  create_duration = "2m"
+}
+
+data "equinix_metal_connection" "NIMF-test" {
+  connection_id     = equinix_metal_connection.metal-connection.id
```

Comment:
Add depends_on = [time_sleep.wait_dl_connection] line in Metal Connection data source block